website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1163,6 +1166,18 @@ $ curl \
 }
 ```
 
+<Note title="Important change to supported versions">
+   As of 1.16.7, 1.17.3 and later, 
+   the <a href="/vault/docs/concepts/billing-start-date">billing start date</a> automatically 
```

Comment:
Tried to maintain consistency by adding a link using [billing start date](/vault/docs/concepts/billing-start-date) but for some reason it does not work here. Therefore used a href here.